tslint.json

```diff
@@ -8,6 +8,7 @@
     "match-default-export-name": false,
     "max-line-length": [true, 80],
     "no-banned-terms": false,
+    "no-relative-imports": false,
```

Comment:
We don't want to allow `no-relative-imports`.
For reference: `//generator/new_parser/parser.ts`.